README.md

```diff
@@ -56,56 +60,95 @@ To get started, create an override configuration for your ESLint, while applying
       "parser": "@graphql-eslint/eslint-plugin",
       "plugins": ["@graphql-eslint"],
       "rules": {
-        ...
+        "@graphql-eslint/require-description": [
+          "error",
+          {
+            "on": [
+              "ObjectTypeDefinition",
+              "InterfaceTypeDefinition",
+              "EnumTypeDefinition",
+              "InputObjectTypeDefinition",
+              "UnionTypeDefinition",
+              "FieldDefinition",
+              "DirectiveDefinition",
+            ],
+          },
+        ],
       }
     }
   ]
 }
 ```
 
-If you are using code files to store your GraphQL schema or your GraphQL operations, you can extend the behaviour of ESLint and extract those, by adding **an additional `override`** that does that extraction processes:
+If your GraphQL definitions are defined only in `.graphql` files and you're only using rules that apply to individual files, you should be good to go :thumbsup:. If you would like to use rules that apply across schema and operation definitions across multiple files, see [here](#using-rules-with-constraints-that-span-the-entire-schema).
 
-```json
+#### Tell ESLint to apply this plugin to GraphQL definitions defined in code files.
+
+If you are defining GraphQL schema or GraphQL operations in code files, you'll want to define an additional override to extend the functionality of this plugin to the GraphQL schema and operations in those files.
+
+```diff
 {
   "overrides": [
-    {
-      "files": ["*.tsx", "*.ts", "*.jsx", "*.js"],
-      "processor": "@graphql-eslint/graphql"
-    },
++    {
++      "files": ["*.tsx", "*.ts", "*.jsx", "*.js"],
++      "processor": "@graphql-eslint/graphql"
++    },
     {
       "files": ["*.graphql"],
       "parser": "@graphql-eslint/eslint-plugin",
       "plugins": ["@graphql-eslint"],
       "rules": {
-        ...
+        "@graphql-eslint/require-description": [
```

Comment:
Added the rule here as well to make this a complete valid value-delivering snippet.

See other comments about the specific rule above.